index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    
+    <h1 id="title">Survey Form</h1>
+     
+    <p id="description">
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Qui exercitationem dolore natus, earum, nesciunt ad consequuntur corporis repellendus nisi, eligendi quos voluptatibus harum iusto dignissimos? Sed deserunt omnis enim dolore.
+    </p>
+    
+    <form action="" id="survey-form">
+     <label for="name" id="name-label">Enter Your Name:</label>
+     <input type="text" id="name" name="username" placeholder="Enter Name" required /> <br/>
+     <label for="email" id="email-label">Enter Your Email Address</label>
+     <input type="email" name="email" id="email" placeholder="Enter Email" required /> <br/>
+     <label for="number" id="number-label">Enter Number (0-100)</label>
+     <input type="number" name="number" id="number" max="100" min="0" placeholder="Enter Number" /> <br/>
+     
+     <div>
+       <select name="dropdown" id="dropdown"> 
+        <option value="male">male</option>
+        <option value="female">female</option>
+       </select>
+      </div>
+
+    <div>
+      <input type="radio" name="team" id="" value="manchesterunited">manchesterunited 
+      <input type="radio" name="team" id="" value="arsenal">arsenal
+      <input type="radio" name="team" id="" value="brazil">brazil
+    </div>
+    <div>
+    <input type="checkbox" name="color" id="" value="red">red 
+    <input type="checkbox" name="color" id="" value="blue">blue
+    <input type="checkbox" name="color" id="" value="green">green
+    </div>
+
+    <div>
+      <textarea name="comment" id="comment" cols="30" rows="10"></textarea>
+    </div>
+    <button type="submit" id="submit">submit</button>
+  </form>
+
+
+    <p>Hello World, we are software developers!!</p>
```

Comment:
Consider wrapping this section of the web page in a ` main ` tag. The semantic tag ` main ` indicates to a web browser and search engine the most important part of your web page